monday/resources/columns.py

```diff
@@ -0,0 +1,13 @@
+from monday.query_joins import update_multiple_column_values_query, create_column
```

Comment:
This resource doesn't exist in the current version of the package, and after testing, neither method works. See comments below. Not sure if you want to fix or just not include @pevner-p2 @rhymiz?